pkg/cli/admin/nodeimage/create.go

```diff
@@ -0,0 +1,54 @@
+package nodeimage
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+func NewCreate(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command {
+	o := NewCreateOptions(streams)
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create an ISO image for booting the nodes to be added to the target cluster",
+		Long: templates.LongDesc(`
+			<TODO>
+		`),
+		Example: templates.Examples(`
+			<TODO>
+		`),
+		Run: func(cmd *cobra.Command, args []string) {
+			kcmdutil.CheckErr(o.Complete(f, cmd, args))
+			kcmdutil.CheckErr(o.Validate())
+			kcmdutil.CheckErr(o.Run(cmd.Context()))
```

Comment:
Most of the time `cmd.Context()` is nil and unfortunately people tend to use it, if it passes (assuming that it is always initialized). I'd recommend removing and once there is a need of a context, it is better to initialize it in `Run` function.